src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--day-fri calendar--month-length-30">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--30">
```
Pay attention to the syntax
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/61bacf1e-64ee-4c68-886e-9063d967b75a)